pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -524,8 +518,29 @@ func (b *MacvtapLibvirtSpecGenerator) generate(domainIface api.Interface) error
 	return nil
 }
 
-func (b *MacvtapLibvirtSpecGenerator) decorateDomainIfaceSpec(domainIface *api.Interface) error {
-	return nil
+func (b *MacvtapLibvirtSpecGenerator) domainIfaceSpec() (*api.Interface, error) {
```

Comment:
likewise; imo, this method is a really good match for having discover in its name, since it reads stuff from netlink.